spec.html

```diff
@@ -34692,14 +34703,17 @@
         <emu-alg>
           1. Assert: _reaction_ is a PromiseReaction Record.
           1. Let _promiseCapability_ be _reaction_.[[Capabilities]].
+          1. Let _type_ be _reaction_.[[Type]].
           1. Let _handler_ be _reaction_.[[Handler]].
-          1. If _handler_ is `"Identity"`, let _handlerResult_ be NormalCompletion(_argument_).
-          1. Else if _handler_ is `"Thrower"`, let _handlerResult_ be Completion{[[Type]]: ~throw~, [[Value]]: _argument_, [[Target]]: ~empty~}.
+          1. If _handler_ is *undefined*, then
+            1. If _type_ is `"Fulfill"`, let _handlerResult_ be NormalCompletion(_argument_).
+            1. Else if _type_ is `"Reject"`, let _handlerResult_ be Completion {[[Type]]: ~throw~, [[Value]]: _argument_, [[Target]]: ~empty~}.
+            1. Assert: _handlerResult_ is defined.
```

Comment:
If you want that, I'd change the second bullet to "Otherwise, _type_ is `"Reject"`. Let _handlerResult_ ..."